authorization-dotnet/BouncyCastleCryptography/BouncyCastleCryptographyTests/AsymmetricEncryptionUnitTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BouncyCastleCryptography.AsymmetricEncryption;
+using Org.BouncyCastle.Crypto;
```

Comment:
Remove unused using and extra whitespace line below. Recheck for all other files in the project.